ReactWindows/ReactNative.Shared/DevSupport/DevSupportManager.cs

```diff
@@ -409,9 +409,12 @@ public void Dispose()
             _devServerHelper.Dispose();
         }
 
-        private async void HandleReloadJavaScript()
+        public async void HandleReloadJavaScript()
```

Comment:
>HandleReloadJavaScript [](start = 26, length = 22)

Note to self  <- why is this public.

Reply:
Makes sense - public callers of this will always be void returning and we need a way of locking the ReactInstanceManager :+1:

---
In reply to: [165944115](https://github.com/Microsoft/react-native-windows/pull/1615#discussion_r165944115) [](ancestors = 165944115)